packages/web-components/fast-components/src/horizontal-scroll/horizontal-scroll.styles.ts

```diff
@@ -110,26 +115,9 @@ export const HorizontalScrollStyles = css`
     }
 
     .content-container {
-        white-space: nowrap;
+        align-items: var(--scroll-align);
+        display: inline-flex;
+        flex-wrap: nowrap;
         position: relative;
     }
-
-    .content-container ::slotted(*) {
-        display: inline-block;
-        white-space: normal;
-        vertical-align: var(--scroll-align);
-    }
```

Comment:
Added a note in @robarbms PR, but I wonder if it might be interesting to apply margin via CSS property to the items. That seems like a better API than *expecting* the items to include it. From a composition standpoint, I typically wouldn't advise to bake external margin into the components. I think an intuitive and elegant approach would be for this to support that w/ a common number (4px) as default value and folks can add *whatever* number they want via CSS property. Thoughts?

Reply:
Added --scroll-item-spacing with a default value of 5px.